modules/workbench/variables.tf

```diff
@@ -36,7 +36,62 @@ variable reporting_dataset_id {
 }
 
 variable reporting_dataset_access {
-  description = "Access grants to the dataset"
-  type        = list(map(string)) # It's not possible to use optional attributes, so this is the tightest type
+  description = "Access list for the reporting dataset"
+  type        = list(map(string))
   default     = []
 }
+
+# TODO(jaycarlton) codegen this top-level variables as the union
+#   of all modules' variables files.
+# List of objects whose values correspond to the google_monitoring_notification_channel
```

Comment:
This stuff doesn't look relevant to the PR - remove? Minimally, should remove commented out code

Reply:
yeah, thought I I needed it, but I guess not.